distribution/receiver/routes/shutdown-route.xml

```diff
@@ -2,8 +2,8 @@
     <route id="shutdown-route">
         <from uri="direct:shutdown-route" />
 
-        <!-- TODO Send a notification to the configured emails -->
-
+      	<toD uri="direct:send-notification-email-route" />
```

Comment:
I think this needs to happen in a try finally block so that in case an error is encountered while sending the email, we are still able to shutdown the application

Reply:
Ok